Documentation/ceph-block.md

```diff
@@ -54,6 +54,20 @@ parameters:
     # Ceph pool into which the RBD image shall be created
     pool: replicapool
 
+    # (optional) mapOptions is a comma-separated list of map options.
+    # For krbd options refer
+    # https://docs.ceph.com/docs/master/man/8/rbd/#kernel-rbd-krbd-options
+    # For nbd options refer
+    # https://docs.ceph.com/docs/master/man/8/rbd-nbd/#options
+    # mapOptions: lock_on_read,queue_depth=1024
+
+    # (optional) unmapOptions is a comma-separated list of unmap options.
+    # For krbd options refer
+    # https://docs.ceph.com/docs/master/man/8/rbd/#kernel-rbd-krbd-options
+    # For nbd options refer
+    # https://docs.ceph.com/docs/master/man/8/rbd-nbd/#options
+    # unmapOptions: force
+
```

Comment:
How do users know if krbd or nbd will be used to know which options to set?

Reply:
The default will be krbd but the user has an option to select the mounter in the storageclass. https://github.com/rook/rook/blob/master/cluster/examples/kubernetes/ceph/csi/rbd/storageclass-test.yaml#L53-L54 based on that he can choose the map/unmap options.

Reply:
I think it would help users if that were documented somewhere. I don't really see that documented explicitly anywhere. The line you link to would be a good place to do that as well as in CRD md docs.